Tester/Framework/Assert.php

```diff
@@ -47,11 +47,11 @@ class Assert
 	 * Checks assertion. Values must be exactly the same.
 	 * @return void
 	 */
-	public static function same($expected, $actual)
+	public static function same($expected, $actual, $message = "")
```

Comment:
I know that `message` is commonly used, but would not it be more accurate to use `description`? 

Btw, use please single quotes where possible.